packages/legend-application/src/components/shared/PackageableElementOptionRenderer.tsx

```diff
@@ -25,12 +25,26 @@ export const getPackageableElementOptionalFormatter = (props?: {
   function PackageableElementOptionLabel(
     option: PackageableElementOption<PackageableElement>,
   ): React.ReactNode {
+    let optionType = '';
+    //if no package, label type system
```

Comment:
let's use the graph to check this instead. You can create methods similar to `isSystemPackageTreeNode` and chuck that in `GraphManagerState`, i.e. `GraphManagerState.isSystemElement(element: PackageableElement): boolean`. FYI there are system elements whose package does not start with `meta`.

Reply:
makes sense, updated!